### PR TITLE
timing: tidy up

### PR DESCRIPTION
**Pull request by MegaApplePi**

### CODE REVIEW

File: wiki/Beatmap_Editor/Timing/en.md [hunk 1]
```diff
@@ -1,41 +1,32 @@
...
+The timing for a [beatmap](/wiki/Beatmaps) consists of the song's offset and BPM (beats per minute).
+The song's offset is the amount of time between the start of an audio file and the first downbeat in the song.
+The song's BPM is essentially how fast a song is.
 
-The point of having correct timing is so that the timeline in the [editor](/wiki/Beatmap_Editor) lines up with the music. If the timing is incorrect, then the timeline does not match up with the song. Therefore, you cannot accurately place hit objects, and it becomes impossible to map correctly to the music.
+The primary reason to use correct timing is that the [timeline](/wiki/timeline) in the [editor](/wiki/editor) lines up with the music.
+If the timing is incorrect, the [hit objects](/wiki/hit objects) on the timeline will not snap to match up with the song.
+Therefore, you cannot accurately place hit objects and it will become nearly impossible to map correctly to the music.
```
Comment (peppy) [inline wiki/Beatmap_Editor/Timing/en.md]: single newlines in markdown are pointless and should be avoided.

File: wiki/Beatmap_Editor/Timing/en.md [hunk 2]
```diff
@@ -1,41 +1,32 @@
...
--   BPM Sensitivity 1: 999,90
--   BPM Sensitivity 2: 999,99
-    -   *9 = random number / 0 = unaffected number.*
+A regular timing section is displayed as a red line in the editor.
+Every map has at least one.
+Some maps change their BPM during the song, however, and require multiple regular timing sections as a result.
 
-**The button «T» may also be used instead of «Tap Here!»**
+Inherited timing sections are displayed as a green line in the editor.
+These do not actually change the song's timing.
```
Comment (peppy) [inline wiki/Beatmap_Editor/Timing/en.md]: you need to learn english grammar. these two sentences should be combined. you should never write things like "What xxx do is ..."

Comment (peppy) [inline wiki/Beatmap_Editor/Timing/en.md]: try `They don't allow changes to a beatmap's timing, but focus on adjusting ...`

File: wiki/Beatmap_Editor/Timing/en.md [hunk 3]
```diff
@@ -1,41 +1,32 @@
...
-Again after the button is pressed enough, BPM sensitivity will change from 0,1 and 2.
+Timing sections allow you to vary the timing across the song.
+There are two types of timing sections: a regular timing section and an inherited timing section.
 
--   BPM Sensitivity 0: 999,00
--   BPM Sensitivity 1: 999,90
--   BPM Sensitivity 2: 999,99
-    -   *9 = random number / 0 = unaffected number.*
+A regular timing section is displayed as a red line in the editor.
+Every map has at least one.
```
Comment (peppy) [inline wiki/Beatmap_Editor/Timing/en.md]: map -> beatmap
has -> must have

File: wiki/Beatmap_Editor/Timing/en.md [hunk 4]
```diff
@@ -1,41 +1,32 @@
...
 
 ![Timing menu](img/Timing_base.jpg "Timing menu")
 
- The timing for a [beatmap](/wiki/Beatmaps) consists of the song's offset and beats per minute (BPM). The song's BPM is essentially how fast the song is. The offset is the amount of time between the start of an mp3 and the first downbeat in the song.
+The timing for a [beatmap](/wiki/Beatmaps) consists of the song's offset and BPM (beats per minute).
+The song's offset is the amount of time between the start of an audio file and the first downbeat in the song.
+The song's BPM is essentially how fast a song is.
 
-The point of having correct timing is so that the timeline in the [editor](/wiki/Beatmap_Editor) lines up with the music. If the timing is incorrect, then the timeline does not match up with the song. Therefore, you cannot accurately place hit objects, and it becomes impossible to map correctly to the music.
+The primary reason to use correct timing is that the [timeline](/wiki/timeline) in the [editor](/wiki/editor) lines up with the music.
```
Comment (peppy) [inline wiki/Beatmap_Editor/Timing/en.md]: "Correct timing is imperative in the beatmapping process to allow placing [objects](/wiki/hit_objects) in time with the beat."

remove following two sentences, unnecessary.

File: wiki/Beatmap_Editor/Timing/en.md [hunk 5]
```diff
@@ -1,41 +1,21 @@
...
 
-![Timing menu](img/Timing_base.jpg "Timing menu")
-
- The timing for a [beatmap](/wiki/Beatmaps) consists of the song's offset and beats per minute (BPM). The song's BPM is essentially how fast the song is. The offset is the amount of time between the start of an mp3 and the first downbeat in the song.
-
-The point of having correct timing is so that the timeline in the [editor](/wiki/Beatmap_Editor) lines up with the music. If the timing is incorrect, then the timeline does not match up with the song. Therefore, you cannot accurately place hit objects, and it becomes impossible to map correctly to the music.
-
-How to time songs
------------------
+_For a guide on timing, see: [How to time songs](/wiki/How_to_time_songs)_
```
Comment (turbedi) [inline wiki/Beatmap_Editor/Timing/en.md]: `/wiki/Guides/Beatmapping/How_to_Time_Songs`

File: wiki/Beatmap_Editor/Timing/en.md [hunk 6]
```diff
@@ -1,41 +1,21 @@
...
+A regular timing section is displayed as a red line in the editor. Every beatmap must have at least one. Some maps change their BPM during the song, however, and require multiple regular timing sections as a result.
 
-Inherited timing sections are displayed as a green line in the editor. These do not actually change the song's timing. What inherited timing sections do is let you change the map's [hitsound](/wiki/Glossary)'s volume, the hitsounds themselves, [slider speed](/wiki/Beatmap_Editor/Slider), etc. for that section without modifying the song's timing (which would happen with a regular timing section).
+Inherited timing sections are displayed as a green line in the editor. They do not allow changes to a beatmap's timing, but focus on adjusting the song's timing. What inherited timing sections do is let you change the beatmap's [hit sound](/wiki/hit_sound)'s volume, the hit sounds themselves, [slider speed](/wiki/Slider), etc. for that section without modifying the song's timing (which would happen with a regular timing section).
```
Comment (turbedi) [inline wiki/Beatmap_Editor/Timing/en.md]: `/wiki/Beatmap_Editor/Slider`. The `hit_sound` link is broken too, but I am not what article it should redirect to.

File: wiki/Beatmap_Editor/Timing/en.md (hunk 5 above)
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/Timing/en.md]: the link is correct when displayed live

File: wiki/Beatmap_Editor/Timing/en.md (hunk 6 above)
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/Timing/en.md]: - The slider link is correct when displayed live.
- The `hit_sound` link should go to the glossary, but it is correct when displayed live.

File: wiki/Beatmap_Editor/Timing/en.md [hunk 7]
```diff
@@ -1,41 +1,21 @@
...
-After a certain number of press, the approximate BPM will be set and the black keys will resonate as «tic toc toc toc» where tic is the start of new time signature (by 4/4 standard). Repeat at least five times and take the average reading. If you are not sure, ask for help in Chat Console or the forum. Upload your beatmap and use the link for request. You may have to do something in return.
-
-Again after the button is pressed enough, BPM sensitivity will change from 0,1 and 2.
+![Timing menu](img/Timing_base.jpg "Timing menu")
 
--   BPM Sensitivity 0: 999,00
--   BPM Sensitivity 1: 999,90
--   BPM Sensitivity 2: 999,99
-    -   *9 = random number / 0 = unaffected number.*
+The timing for a [beatmap](/wiki/Beatmaps) consists of the song's offset and BPM (beats per minute). The song's offset is the amount of time between the start of an audio file and the first downbeat in the song. The song's BPM is essentially how fast a song is.
```
Comment (TPGPL) [inline wiki/Beatmap_Editor/Timing/en.md]: more like `The beatmap's timing` or `The timing of a beatmap`

File: wiki/Beatmap_Editor/Timing/en.md [hunk 8]
```diff
@@ -1,41 +1,21 @@
...
+## Setup
 
-![The Timing Setup dialog showing a map with a fluctuating BPM and sampleset changes.](img/TimingSetup.jpg "The Timing Setup dialog showing a map with a fluctuating BPM and sampleset changes.")
+![Screenshot of the "Timing and Control Points" window](img/TimingSetup.jpg "The timing setup dialog shows the beatmap's fluctuating BPM and sampleset changes")
 
-Timing sections allow you to vary the timing across the song. There are two types of timing sections, a regular timing section and an inherited timing section.
+Timing sections allow you to vary the timing across the song. There are two types of timing sections: a regular timing section and an inherited timing section.
 
-A regular timing section is displayed as a red line in the editor. Every map has at least one. Some maps change their BPM during the song, however, and require multiple regular timing sections as a result.
+A regular timing section is displayed as a red line in the editor. Every beatmap must have at least one. Some maps change their BPM during the song, however, and require multiple regular timing sections as a result.
```
Comment (TPGPL) [inline wiki/Beatmap_Editor/Timing/en.md]: However, some beatmaps change their BPM during the song and require multiple regular timing sections as a result.
don't use the `map` word i suppose

Comment (MegaApplePi) [reply wiki/Beatmap_Editor/Timing/en.md]: 😕 the entire article doesn't use `map` anywhere

File: wiki/Beatmap_Editor/Timing/en.md (hunk 6 above)
Comment (TPGPL) [inline wiki/Beatmap_Editor/Timing/en.md]: 1. Inherited timing section is displayed as a green line
2. They do not allow changes to the beatmap's timing (a -> the, i think?)
3. slider speed into slider velocity?

File: wiki/Beatmap_Editor/Timing/en.md [hunk 9]
```diff
@@ -1,41 +1,21 @@
...
 
-Inherited timing sections are displayed as a green line in the editor. These do not actually change the song's timing. What inherited timing sections do is let you change the map's [hitsound](/wiki/Glossary)'s volume, the hitsounds themselves, [slider speed](/wiki/Beatmap_Editor/Slider), etc. for that section without modifying the song's timing (which would happen with a regular timing section).
+Inherited timing sections are displayed as a green line in the editor. They do not allow changes to a beatmap's timing, but focus on adjusting the song's timing. What inherited timing sections do is let you change the beatmap's [hit sound](/wiki/hit_sound)'s volume, the hit sounds themselves, [slider speed](/wiki/Slider), etc. for that section without modifying the song's timing (which would happen with a regular timing section).
 
-These two types of timing sections are also commonly referred to as "red offsets" and "green offsets" respectively.
+These two types of timing sections are also commonly referred to as _red offsets_ and _green offsets_ respectively.
```
Comment (TPGPL) [inline wiki/Beatmap_Editor/Timing/en.md]: These -> Those

referred to as -> called

File: wiki/Beatmap_Editor/Timing/en.md (hunk 8 above)
Comment (TPGPL) [reply wiki/Beatmap_Editor/Timing/en.md]: `Some maps change their BPM`

**Pull request merged**